app/frontend/constants.ts

```diff
@@ -18,3 +20,9 @@ export enum SubTabs {
   BALANCE = 'Balance',
   SHELLEY_BALANCES = 'Shelley Balances',
 }
+
+export const AuthMethodNames = {
+  [AuthMethodEnum.MNEMONIC]: 'Mnemonic',
```

Comment:
I would directly put this into the getAuthMethodName function and leverage typescript to ensure no auth method is left out